src/mono/mono/mini/CMakeLists.txt

```diff
@@ -84,6 +84,9 @@ set(mini_common_sources
     aot-compiler.h
     aot-compiler.c
     aot-runtime.c
+    # metadata_amodule->method_ref_to_method
+    ../../../native/containers/dn-simdhash.c
```

Comment:
You don't need it here. it will flow in from metadata/CMakeFiles.txt from `imported_native_sources` which is part of `metadata_sources` which gets linked into the same binaries as mini_common_sources

I think probably for these kinds of third-party things it's better to just list them all in mini/CMakeLists.txt rather than in sgen/ utils/ metadata/

Reply:
It wasn't building locally without doing this, though.

Reply:
Moving it all into mini/ seems to have done the trick, thanks!